app/controllers/api/v1/npq_applications_controller.rb

```diff
@@ -26,19 +26,15 @@ def show
       end
 
       def reject
-        if NPQ::Reject.call(npq_application:)
-          render json: json_serializer_class.new(npq_application).serializable_hash
-        else
-          render json: { errors: Api::ErrorFactory.new(model: npq_application).call }, status: :bad_request
-        end
+        service = NPQ::RejectApplication.new(npq_application:)
```

Comment:
I would not have tied the action to the namespace but rather would have added classes as such:
```ruby
module NPQ
  module Application
    class Reject; end
    class Accept; end
  end
end
```